modules/cnv-live-migration-limits-reftable.adoc

```diff
@@ -0,0 +1,37 @@
+
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-live-migration-limits.adoc
+
+[id="cnv-live-migration-limits-ref_{context}"]
+= Cluster-wide live migration limits and timeouts
+
+.Migration parameters
+|===
+|Parameter |Description |Default
+
+|`parallelMigrationsPerCluster`
+|Number of migrations running in parallel in the cluster. 
+|5
+
+|`parallelOutboundMigrationsPerNode`
+|Maximum number of outbound migrations per node.
+|2
+
+|`bandwidthPerMigration`
+|Bandwidth limit of each migration, in MiB/s.
+|64Mi
+
+|`completionTimeoutPerGiB`
+|The migration will be aborted if it has not completed in this time, in seconds 
+per GiB of memory. For example, a VMI with 6GiB memory will timeout if it has 
```

Comment:
I'd mention the value of completionTimeoutPerGiB here. It's implied that it's the default value, but I've seen people file bugs about similar issues.

I'd also consider removing the future tense. I think it's clear enough in present tense.